### ISSUE TITLE
Grandia II

### ISSUE THREAD

**Issue by Brandonheat8** [Rinnegatamante/flycast-vita-Compatibility#41]
# Game summary
- Game name: Grandia II (US)
- Game version: v1.001

# flycast summary
- Version: Latest version with SMC

# Issues
- Is now playable from pressing start (before you needed a save file because the opening cutscene would crash the system) Menus run from 22-30FPS mostly a full 30FPS, cutscenes run anywhere from 8-19FPS mostly on the 15-17 range depending on load, Gameplay runs from 11-17FPS, and Battles run from 20-28FPS which is fantastic and pretty much full speed depending on what's going on, was able to complete a battle and save the game without any crashes.

# Screenshots
![PS_App_20210102_021358](https://user-images.githubusercontent.com/76890866/103471605-8552b300-4d3f-11eb-9e6f-1750a7aef198.jpeg)
![PS_App_20210102_021406](https://user-images.githubusercontent.com/76890866/103471610-90a5de80-4d3f-11eb-890e-8a9246264dac.jpeg)
![PS_App_20210102_021658](https://user-images.githubusercontent.com/76890866/103471612-9996b000-4d3f-11eb-8098-65b0be0e215d.jpeg)
![PS_App_20210102_021410](https://user-images.githubusercontent.com/76890866/103471616-a1eeeb00-4d3f-11eb-9304-dcb4bdd00abf.jpeg)

# Recommended labels

- B - Playable

**Comment by Ted-bissada** [comment]
are you using any special settings or something because when i try and start a new game on the self modifying code check version using a gdi rom i hit select on the menu then after a now  loading screen it just black screens forever on the non self modifying version it just restarts to the dreamcast splash and then returns to menu

**Comment by Brandonheat8** [comment]
> are you using any special settings or something because when i try and start a new game on the self modifying code check version using a gdi rom and using the provided settings file when i hit select on the menu then after a now loading screen it just black screens forever on the non self modifying version it just restarts to the dreamcast splash and then returns to menu

Well I am overclocked to 500Mhz, also using GDI. I'll re test it again.

**Comment by Ted-bissada** [comment]
i ended up using my pc to generate a save for the first checkpoint and it loads fine from there but i still cannot start a new game for some reason

**Comment by Brandonheat8** [comment]
> i ended up using my pc to generate a save for the first checkpoint and it loads fine from there but i still cannot start a new game for some reason

You're right I just tried doing a new game again and yup stuck on black screen, I went back to an older build dated Jan 1st, 2020 and the opening does work. It was the last build they made right before adding the custom live area.

![PS_App_20210103_053041](https://user-images.githubusercontent.com/76890866/103472400-35c5b480-4d4a-11eb-94dc-f63079426a4c.jpeg)

**Comment by MrWizardJM** [comment]
you need to run in accuracy
